task_processing/plugins/persistence/dynamodb_persistence.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+import decimal
+
+import boto3.session as bsession
+from boto3.dynamodb.conditions import Key
+from pyrsistent import thaw
+
+from task_processing.interfaces.persistence import Persister
+
+
+class DynamoDBPersister(Persister):
+    def __init__(self, table_name, endpoint_url=None, session=None):
+        self.table_name = table_name
+        if not session:
+            session = bsession.Session()
+        self.ddb_client = session.client(
+            service_name='dynamodb',
+            endpoint_url=endpoint_url,
+        )
+        self.table = session.resource(
+            endpoint_url=endpoint_url,
+            service_name='dynamodb'
+        ).Table(table_name)
+
+    def read(self, task_id, comparison_operator='EQ'):
```

Comment:
I can imagine that if paasta encodes the task_id as service.instance.uuid then we'll do ``BEGINS_WITH`` to find all the tasks for a given service.instance